docs/Rakefile

```diff
@@ -78,7 +78,7 @@ task :adoc_tar => [:init, :adoc_variables, :markdown_to_asciidoc] do
       file_list.each { |f|
         tar.add_file(f, 0444) { |io|
           if File.extname(f) == ".adoc" then
-            adoc_content = File.read(f)
+            adoc_content = File.read(f, encoding: "ISO8859-1:utf-8")
```

Comment:
Which problem? The error in the logs was about 'clean'. I guess _rm_ fails if the file does not exist which would be the case when running _clean_ prior to any other target. 

How does encoding play in here. TBH, this does not look right.